config/target.json

```diff
@@ -0,0 +1,5 @@
+{
+  "generate_icicle": false,
+  "openstack_image_format": "qcow2",
+  "rhevm_image_format": "qcow2"
```

Comment:
Hmm, that is weird. See:

https://github.com/redhat-imaging/imagefactory/blob/master/imagefactory#L180

Here you see all 3 command line options that add/remove configuration being merged into `ApplicationConfig().configuration`. I will do some tests tonight and see what is happening. Did you try it out, or were you reasoning about the code?